### DISCUSSION TITLE
DNS with sort of Round Robin functionalities and Health Checks.

### DISCUSSION THREAD

**Discussion by NeoPlayer13** [0xERR0R/blocky#1443] [Ideas]
Is there any posibility of adding to Blocky the capability of pointing a DNS record to two or more IP's and make it capable of selecting which one to use by using a health check method or by runing a script?

**Comment by kwitsch** [comment]
Technically possible but that's most likely not what you really want since it would mess with the way modern browsers and we apps work. 

It's more likely you want a combination of haproxy with keepalived or something similar.
